src/paths/pages/Distribution.tsx

```diff
@@ -9,7 +10,8 @@ const Distribution = () => {
         backgroundColor: "#E7DCD5",
         flexGrow: 1
       }}>
-        Distribution
+        
+        <PackagesList> </PackagesList>
```

Comment:
When component doesn't have any children then you can use shorter syntax: `<PackageList />`